src/General.php

```diff
@@ -12,6 +12,57 @@ public function __construct()
         $this->files = $_FILES;
     }
 
+    public function getCssNameFromDb() {
+      return $this->id = $_COOKIE['theme'] ;
+    }
+
+    public function getNameFromDb() {
+      $id = substr($_COOKIE['user'],2);
+      $result = $this->getConnection()->query("SELECT * FROM users WHERE id = '$id' ");
+      while ($row = mysqli_fetch_row($result)) {
+        $data[] = $row;
+        }
+      foreach ($data as $key => $value) {
+        $name = $value[1];
+      }
+      return $this->name = $name;
+    }
+
+    public function displayAlert() {
+      if ($this->checkAlertIsAccepted() == false) {
+       echo '<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+                <script src="/Warsztaty_4/src/views/js/js.cookie.js"></script>
+                <script src="/Warsztaty_4/src/views/js/alert.js"></script>' .
+                '<div id="myModal" class="reveal-modal">
+                      <h3>Strona tylko dla osób dorosłych</h3>
+                			<p>Zamykając powiadomienie potwierdzasz swoją pełnoletność</p>
+                			<a class="close-reveal-modal">&#215;</a>
+                		</div>';
+      }
+    }
+
+    public function checkAlertIsAccepted() {
+      if (isset($_COOKIE['alert'])) {

```

Comment:
to samo co wyżej - $_COOKIE przekazana w konstruktorze. 

Druga sprawa: ten else jest zupełnie zbędny.

```php
return isset($this->cookie['alert']);
```